### ISSUE TITLE
bcrypt has no attribute 'checkpw'

### ISSUE THREAD

**Issue by Debetux** [SirCmpwn/sr.ht#6]
Hi,

I played a bit with your app, and hu, I got this error. I tried to take a look at the bcrypt module doc, but I got nothing. I'll take a look later.
```
  File "/Users/.../.../.../sr.ht/srht/blueprints/html.py", line 94, in login
    if not bcrypt.checkpw(password, user.password):
AttributeError: 'module' object has no attribute 'checkpw'
```

Thanks,
Lancelot.

**Comment by Addisonbean** [comment]
Did anyone ever figure this out? I'm having this same issue.

**Comment by SirCmpwn** [comment]
It was fixed by #8, which is updated on https://gogs.sr.ht/SirCmpwn/sr.ht.
